site/content/en/docs/Reference/API/Kustomization File/namePrefix.md

```diff
@@ -4,7 +4,18 @@ linkTitle: "namePrefix"
 type: docs
 weight: 11
 description: >
-    Prepends the value to the names of all resources and references.
+    Add prefixes to the names of all resources.
 ---
+`apiVersion: kustomize.config.k8s.io/v1beta1`
 
-The Tasks section contains examples of how to use [`namePrefix`](/docs/tasks/namespaces_and_names/).
+See the [Tasks section] for examples of how to use `namePrefix`.
+
+### namePrefix
+Add prefixes to the names of all resources.
+
+* **namePrefix** (string)
+
+    NamePrefix will add a prefix the names of all resources mentioned in the Kustomization file including generated resources such as ConfigMaps and Secrets.
```

Comment:
```suggestion
    NamePrefix will add a prefix the names of all resources mentioned in the Kustomization file including generated resources such as ConfigMaps and Secrets.
```
nit.
prefix `to` the names

Reply:
Thanks for your review! I updated this per your suggestion. Resolving this comment.